pkg/base1/test-locale.js

```diff
@@ -227,4 +227,9 @@ document.addEventListener("DOMContentLoaded", event => {
 
     /* Ready to go */
     QUnit.start();
-});
+}
+
+if (document.readyState == "loading")
+    document.addEventListener("DOMContentLoaded", init);
+else
+    init();
```

Comment:
Hmm, is the `DOMContentLoaded` handler not called if the document is already "ready"? We use the old pattern a lot, if that is broken it should be replaced everywhere?

Reply:
This is a consequence of the new top-level "await" in qunit-tests.js.  It delays the execution of the rest of the script.